app/models/insights_client_report_status.rb

```diff
@@ -3,12 +3,9 @@ class InsightsClientReportStatus < HostStatus::Status
 
   REPORTING             = 0 # host_registration_insights = true & getting data
   NO_REPORT             = 1 # host_registration_insights = true & not getting data
-  NOT_MANAGED           = 2 # host_registration_insights = false
-  NOT_MANAGED_WITH_DATA = 3 # host_registration_insights = false & getting data
```

Comment:
@stejskalleos can you also take a look? thanks

Reply:
We should remove comments `host_registration_insights = true`, since `host_params` are being ignored